sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -231,58 +231,59 @@ else if (protocolMsg.getBulkData() != null) {
-			if (data.length > getMtu(sessionType)) {
+			if (data != null) {
+				if (data.length > getMtu(sessionType)) {
 
-				messageID++;
-	
-				// Assemble first frame.
-				Long mtu = getMtu(sessionType);
-				int frameCount = new Long(data.length / mtu).intValue();
-				if (data.length % mtu > 0) {
-					frameCount++;
-				}
-				//byte[] firstFrameData = new byte[HEADER_SIZE];
-				byte[] firstFrameData = new byte[8];
-				// First four bytes are data size.
-				System.arraycopy(BitConverter.intToByteArray(data.length), 0, firstFrameData, 0, 4);
-				// Second four bytes are frame count.
-				System.arraycopy(BitConverter.intToByteArray(frameCount), 0, firstFrameData, 4, 4);
-
-				SdlPacket firstHeader = SdlPacketFactory.createMultiSendDataFirst(sessionType, sessionID, messageID, getMajorVersionByte(),firstFrameData,protocolMsg.getPayloadProtected());
-				firstHeader.setPriorityCoefficient(1+protocolMsg.priorityCoefficient);
-				//Send the first frame
-				handlePacketToSend(firstHeader);
-				
-				int currentOffset = 0;
-				byte frameSequenceNumber = 0;
-
-				for (int i = 0; i < frameCount; i++) {
-					if (i < (frameCount - 1)) {
-	                     ++frameSequenceNumber;
-	                        if (frameSequenceNumber ==
-	                                SdlPacket.FRAME_INFO_FINAL_CONNESCUTIVE_FRAME) {
-	                            // we can't use 0x00 as frameSequenceNumber, because
-	                            // it's reserved for the last frame
-	                            ++frameSequenceNumber;
-	                        }
-					} else {
-						frameSequenceNumber = SdlPacket.FRAME_INFO_FINAL_CONNESCUTIVE_FRAME;
-					} // end-if
+					messageID++;
+		
+					// Assemble first frame.
+					Long mtu = getMtu(sessionType);
+					int frameCount = new Long(data.length / mtu).intValue();
+					if (data.length % mtu > 0) {
+						frameCount++;
+					}
+					//byte[] firstFrameData = new byte[HEADER_SIZE];
+					byte[] firstFrameData = new byte[8];
+					// First four bytes are data size.
+					System.arraycopy(BitConverter.intToByteArray(data.length), 0, firstFrameData, 0, 4);
+					// Second four bytes are frame count.
+					System.arraycopy(BitConverter.intToByteArray(frameCount), 0, firstFrameData, 4, 4);
+
+					SdlPacket firstHeader = SdlPacketFactory.createMultiSendDataFirst(sessionType, sessionID, messageID, getMajorVersionByte(),firstFrameData,protocolMsg.getPayloadProtected());
+					firstHeader.setPriorityCoefficient(1+protocolMsg.priorityCoefficient);
+					//Send the first frame
+					handlePacketToSend(firstHeader);
 					
-					int bytesToWrite = data.length - currentOffset;
-					if (bytesToWrite > mtu) {
-						bytesToWrite = mtu.intValue();
+					int currentOffset = 0;
+					byte frameSequenceNumber = 0;
+
+					for (int i = 0; i < frameCount; i++) {
+						if (i < (frameCount - 1)) {
+		       				++frameSequenceNumber;
+		         			if (frameSequenceNumber == SdlPacket.FRAME_INFO_FINAL_CONNESCUTIVE_FRAME) {

```

Comment:
This `if` statement still needs to be tabbed in one more time

Reply:
This could be a git issue.  It looks pretty good on my editors.
<img width="827" alt="screen shot 2018-02-13 at 12 49 01 pm" src="https://user-images.githubusercontent.com/8460594/36173167-e651ed00-10bc-11e8-8c08-176b29f8acbb.png">
![screen shot 2018-02-13 at 12 48 28 pm](https://user-images.githubusercontent.com/8460594/36173170-e91232b6-10bc-11e8-96ac-9194e73eced0.png)

Reply:
Attatched is the raw view. It looks like there are spaces and tabs mixed in and that is likely causing the issue. ![image](https://user-images.githubusercontent.com/6579474/36173387-d94cea6e-10d6-11e8-838e-a1afdd6383da.png)